### PR TITLE
KFSPTS-7165 implement upload suppliers

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

**Comment by jhulslander** [comment]
This is ready for review I believe

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
switched reviewer from Bryan to Nancy

File: src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksKfsServiceImpl.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
 import edu.cornell.kfs.paymentworks.service.PaymentWorksAchConversionService;
 import edu.cornell.kfs.paymentworks.service.PaymentWorksKfsService;
 import edu.cornell.kfs.paymentworks.service.PaymentWorksNewVendorConversionService;
-import edu.cornell.kfs.paymentworks.service.PaymentWorksSupplierConversionService;
+import edu.cornell.kfs.paymentworks.service.PaymentWorksUploadSupplierService;
```
Comment (nkimble) [inline src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksKfsServiceImpl.java]: I am not seeing where this imported service is used. Am I missing something?

File: src/main/java/edu/cornell/kfs/paymentworks/service/PaymentWorksUploadSupplierService.java [hunk 2]
```diff
@@ -24,9 +24,18 @@
 	String createSupplierUploadFile(List<PaymentWorksSupplierUploadDTO> paymentWorksSupplierUploadList,String directoryPath);
 	
 	/**
-	 * Deletes the file passed in.
-	 * @param fileName
+	 * Uploads approved suppkiers to PaymentWorks
 	 */
-	void deleteSupplierUploadFile(String fileName);
```
Comment (nkimble) [inline src/main/java/edu/cornell/kfs/paymentworks/service/PaymentWorksUploadSupplierService.java]: Curious.  How will the removal of bad or unneeded files that were uploaded be dealt with without this method?

Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/paymentworks/service/PaymentWorksUploadSupplierService.java]: I spoke with Purchasing about the deleting of upload files.  There are a couple of scenarios where a file might not get uploaded to payment works, so they opted to keep backups on our batch server.
I created another user story in the backlog to figure out how we want to clean up the upload files.  Sandy noticed that user story and she told me about the nightly job that deleting files older than ~90 (I think) days.  So we are going to make use of that.

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
this is ready for review again

**Comment by bhutchinson** [comment]
One question I have - I see this PR has 10 commits - is the idea that once it is merged the squash merge option would be used so we only end up with one commit on the post6-upgrade branch?

**Comment by jhulslander** [comment]
@bhutchinson Yes, squash into post-6-upgrade is what I expected.  Maybe I misunderstood something, I thought that is what we had agreed to start trying in the last MOPs.

**Comment by jhulslander** [comment]
thank you

**Pull request merged**